### ISSUE TITLE
Ruby Version 3 Never Shows Up

### ISSUE THREAD

**Issue by Oglopf** [rbenv/ruby-build#2109]
```
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.1 LTS
Release:	22.04
Codename:	jammy
rbenv version: 1.1.2
``` 

Followed the instructions as given for ubuntu but when I issue the `rbenv install 3.1.2` command it gives the error:
```
ruby-build: definition not found: 3.1.2
```
And looking through the list from `rbenv install --list` it jumps from  ruby `2.8` to `jruby-1.5.6` and others. 
```
rbenv install --list
...
2.7.0
2.7.1
2.8.0-dev
jruby-1.5.6
jruby-1.6.3
...
```

Am I missing something to get any `3.x.x` choices?

**Comment by mislav** [comment]
Hi, you can inspect your ruby-build version with:
```
rbenv install --version
```
I suspect that your version is very old. I don't know how you installed it, but here are some instructions on installing the latest one: https://github.com/rbenv/ruby-build/wiki#updating-ruby-build

**Comment by Oglopf** [comment]
Ok. I did build off the GitHub instructions as instructed and I've done everything in the instructions but no luck.

What did get things working was the old trick of reverting to `openssl` version 1 and setting the environment variable in my `bashrc` to use that version of `openssl`. 

I don't have time to go track down what's going on, but I want to reiterate I followed those Git instructions "for the latest build" and this was my end state.
